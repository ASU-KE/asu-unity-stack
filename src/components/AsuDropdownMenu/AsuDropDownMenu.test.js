import React from 'react'
import { render} from '@testing-library/react'
import ASUDropdownMenu from './AsuDropdownMenu'

describe('<AsuDropdownMenu /> spec', () => {
    it('renders the component', () => {
      const container = render(<ASUDropdownMenu />, {title: 'ASU Home'});
      expect(container.firstChild).toMatchSnapshot()
     })
});