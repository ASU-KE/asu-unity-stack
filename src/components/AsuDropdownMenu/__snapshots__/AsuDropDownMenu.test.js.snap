// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AsuDropdownMenu /> spec renders the component 1`] = `undefined`;
